packages/contact-importer/src/importer.spec.js

```diff
@@ -1,3 +1,4 @@
+const sendgrid = require('sendgrid');
```

Comment:
Why?

Reply:
The test file uses sendgrid, but it wasn't in the file. This was required to get it to run and pass.

Reply:
I would also add this script shortcut in main package json:
```
{
  // ...
  scripts: {
    // ... 
    "test:contact": "babel-node ./node_modules/istanbul/lib/cli cover ./node_modules/mocha/bin/_mocha \"packages/contact-importer/**/*.spec.js\"",
  }
}
```

Reply:
Good call, I'll get that added shortly.